### PR TITLE
Issue #12720: Keep workflows inputs version consistent

### PR DESCRIPTION
**Pull request by stoyanK7**
Part of #12720
The purpose of this PR is to divide #12741 into smaller chunks
All workflows use 
```yml
on:
  workflow_dispatch:
    inputs:
      version:
```
apart from the file in this PR.

### CODE REVIEW

File: .github/workflows/copy-site-to-sourceforge.yml [hunk 1]
```diff
@@ -6,17 +6,22 @@ name: "R: Copy Site to SourceForge"
 on:
   workflow_dispatch:
     inputs:
-      release-version:
+      version:
         description: 'Release Version without (-SNAPSHOT)'
         required: true
+  workflow_call:
+    inputs:
```
Comment (romani) [inline .github/workflows/copy-site-to-sourceforge.yml:13]: Better to keep workflow_call in workflow introduction PR.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .github/workflows/copy-site-to-sourceforge.yml (hunk 1 above)
Comment (stoyanK7) [reply .github/workflows/copy-site-to-sourceforge.yml:13]: So, close this PR then?

Comment (romani) [reply .github/workflows/copy-site-to-sourceforge.yml:13]: Keep changes in input syntax, and let's merge

Comment (stoyanK7) [reply .github/workflows/copy-site-to-sourceforge.yml:13]: Wait, I dont get what you mean. So just leave changes in `copy-site-to-sourceforge.yml` where `github.event.inputs.release-version` -> `inputs.version` and `release-version` -> `version`?

Comment (stoyanK7) [reply .github/workflows/copy-site-to-sourceforge.yml]: Done. Hopefully that's what you meant.

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**